clinica/iotools/converters/oasis3_to_bids/oasis3_utils.py

```diff
@@ -248,9 +248,29 @@ def write_to_tsv(dataframe: DataFrame, buffer: Union[PathLike, BinaryIO]) -> Non
 
 
 def install_nifti(sourcedata_dir: PathLike, bids_filename: PathLike) -> None:
+    from pathlib import Path
+
     from fsspec.implementations.local import LocalFileSystem
 
     fs = LocalFileSystem(auto_mkdir=True)
+
```

Comment:
I'd move the whole code section below the nifti install part, since all this is optional / conditional. Let's also add a comment explaining what this section does, i.e. install the sidecar BIDS files if present.

Reply:
I still have to add an explanation